packages/session-replay-browser/src/config.ts

```diff
@@ -32,7 +34,10 @@ export class SessionReplayConfig extends Config implements ISessionReplayConfig
     this.apiKey = apiKey;
     this.sampleRate = options.sampleRate || DEFAULT_SAMPLE_RATE;
 
-    this.deviceId = options.deviceId;
-    this.sessionId = options.sessionId;
+    if (options.sessionId && options.deviceId) {
```

Comment:
this seems like we would fail silently. should log an error if these are no available?

Reply:
Added a check!